components/Header/index.tsx

```diff
@@ -3,16 +3,15 @@ import React from 'react'
 
 const Header = ({ user }: any) => (
   <AppBar position="static">
-    <div className="bg-secondary pl-10  p-5 flex justify-between">
+    <div className="bg-secondary pl-10  p-5 flex justify-between h-26">
       <img src="./images/wallora-logo-170x53.jpeg" alt="This is Wallora logo." />
-      <span className="text-primary">This is header</span>
       {user ? (
         <Button variant="contained" color="primary" href="/api/auth/logout">
```

Comment:
reduce default button height too like a standard material UI button. The current Login button seems taller than regular buttons